platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1456,15 +1457,16 @@ public void setupWithNoData() {
      * {@inheritDoc}
      */
     @Override
-    public LearnerTreeView<Long> buildLearnerView(final ReconnectConfig reconnectConfig) {
+    public LearnerTreeView<Long> buildLearnerView(
+            final ReconnectConfig reconnectConfig, final ReconnectMapStats mapStats) {
```

Comment:
missing nullity annotation for new param

Reply:
added